tests/conftest.py

```diff
@@ -127,6 +127,7 @@ def fix_resume_path(request):
     # pylint: disable=protected-access
     prev = resuming._RESUME_DIR
     resuming._RESUME_DIR = tempfile.mkdtemp()
+    resuming.is_db_initialized = False
```

Comment:
Please add a line asserting its existence prior to assignment. Otherwise it will break if (and when) you rename it